tests/lib/config/config-initializer.js

```diff
@@ -232,25 +232,25 @@ describe("configInitializer", function() {
                     format: "JSON",
                     commonjs: false
                 };
-                origLog = console.log;
-                console.log = function() {}; // necessary to replace, because of progress bar
+
+                sandbox = sinon.sandbox.create();
+                sandbox.stub(console, "log"); // necessary to replace, because of progress bar
+
                 process.chdir(fixtureDir);
+
                 try {
                     config = init.processAnswers(answers);
                     process.chdir(originalDir);
                 } catch (err) {
 
-                    // if processAnswers crashes, we need to be sure to restore cwd and console.log
-                    console.log = origLog;
+                    // if processAnswers crashes, we need to be sure to restore cwd
                     process.chdir(originalDir);
                     throw err;
+                } finally {
+                    sandbox.restore();  // restore console.log()
                 }
             });
 
-            beforeEach(function() {
-                console.log = origLog;
-            });
```

Comment:
You could also try `node_modules/.bin/mocha
tests/lib/config/config-initializer.js`.

On Thu, Apr 14, 2016 at 2:20 PM Kevin Partington <notifications@github.com>
wrote:

> In tests/lib/config/config-initializer.js
> <https://github.com/eslint/eslint/pull/5849#discussion_r59765989>:
>
> >                  }
> >              });
> >
> > -            beforeEach(function() {
> > -                console.log = origLog;
> > -            });
>
> I ran npm test and didn't see any strange output, is that sufficient? Or
> is there more I need to do to check this case out?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/pull/5849/files/815ef6069c646a90e3202c6d71d627ac2c14c771#r59765989>
>